src/legacy/core_plugins/vis_type_timelion/public/helpers/tick_formatters.test.ts

```diff
@@ -17,124 +17,123 @@
  * under the License.
  */
 
-import expect from '@kbn/expect';
-import { tickFormatters } from '../../services/tick_formatters';
+import { tickFormatters } from './tick_formatters';
```

Comment:
Great, thanks for already converting those tests!

Reply:
And after removing old code, `vis_type_timelion` doesn't need `tickFormatters` any more. So I moved it back to `timelion`.